deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -4,28 +4,31 @@ This repo contains an IaC CDK solution for deploying an OpenSearch Service Domai
 
 ### Getting Started
 
+#### Project required setup
+
+It is necessary to run `npm install` within this current directory to install required packages that this app and CDK need for operation.
+
 #### First time using CDK?
 
-You can install the CDK CLI tool by running:
+If this is your first experience with CDK, follow the steps below to get started:
+
+1- Install the **CDK CLI** tool by running:
 ```
 npm install -g aws-cdk
 ```
 
-You then will need to configure the desired [AWS credentials](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites), as these will dictate the region and account used for deployment.
+2- Configure the desired **[AWS credentials](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites)***, as these will dictate the region and account used for deployment.
+
+3- Set the `CDK_DEPLOYMENT_STAGE` environment variable to assist in naming resources and preventing collisions. Typically, this would be set to a value such as `dev`, `gamma`, `Wave1`, `PROD` and will be used to distinguish AWS resources for a given region and deployment stage. For example the CloudFormation stack may be named like `OSServiceDomain-dev-us-east-1`. This stage environment variable should only be used for the disambiguation of user resources.
```

Comment:
This env variable is something we've created and is specific to our repo. CDK users likely have not seen it, so should we move it to the project specific setup section?

Reply:
Oh I see, moving this back in the next commit. Thank you Tanner